Playbooks/AS-Slack-Integration/azuredeploy.json

```diff
@@ -0,0 +1,505 @@
+                        },
+                        "Initialize_Variable_-_Description": {
+                            "runAfter": {
+                                "Condition_-_Check_and_Set_Severity_Icon": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Description",
+                                        "type": "string"
+                                    }
+                                ]
+                            }
+                        },
+                        "Initialize_Variable_-_Entities": {
+                            "runAfter": {
+                                "Condition_-_Check_if_Description_Exists": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Entities",
+                                        "type": "string"
+                                    }
+                                ]
+                            }
+                        },
+                        "Initialize_Variable_-_Severity_Icon": {
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Severity Icon",
+                                        "type": "string",
+                                        "value": ""
+                                    }
+                                ]
+                            },
+                            "runAfter": {
+                                "Initialize_variable__-_OAuthToken": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable"
+                        },
+                        "Initialize_variable_-_ChannelId": {
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "ChannelId",
+                                        "type": "string",
+                                        "value": "[parameters('ChannelId')]"
+                                    }
+                                ]
+                            },
+                            "runAfter": {},
+                            "type": "InitializeVariable"
+                        },
+                        "Initialize_variable__-_OAuthToken": {

```

Comment:
No need to separately have a variable for **OAuthToken**. Data is already present in parameter and can be used from there. This would avoid displaying secret information in plain text.

Reply:
This is resolved